rbac/server/api/users.py

```diff
@@ -163,10 +165,22 @@
 async def delete_user(request, next_id):
     """Delete a specific user by next_id."""
 
-    txn_key, txn_user_id = await utils.get_transactor_key(request)
-    batch_list = DeleteUser().batch_list(
-        signer_keypair=txn_key, signer_user_id=txn_user_id, next_id=next_id
+    user_delete = DeleteUser()
+    txn_list = []
+    txn_key, _ = await utils.get_transactor_key(request)
+    txn_list = await create_delete_role_owner_txns(txn_key, next_id, txn_list)
+    message = user_delete.make(signer_keypair=txn_key, next_id=next_id)
```

Comment:
We can add a new function to the user_transactions.py file by that uses lines 168 and 172-177 from this file as this is a similar to create_delete_role_owner_txns().